LayoutTests/http/tests/security/clipboard/clipboard-file-access.html

```diff
@@ -126,6 +126,7 @@
 
 function runTest()
 {
+    testRunner.waitUntilDone();
     debug("Dragging no files should return an empty file list (arbitrary implementation detail):");
     testDraggingFiles([]);
```

Comment:
I think you need to await these too.

Reply:
This test is actually wk1 only, I'll undo these changes.